sweetest/src/main/java/com/mysugr/sweetest/framework/dependency/DependencyManager.kt

```diff
@@ -20,6 +20,10 @@ class DependencyManager(setupHandlerReceiver: (DependencySetupHandler) -> Unit)
     private val initializerContext = object : DependencyInitializerContext() {
         override fun <T : Any> instanceOf(clazz: KClass<T>): T {
             val configuration = configurations.getAssignableTo(clazz)
+                ?: throw DependencyConfigurations.NotFoundException(
```

Comment:
why is that necessary? 🤔

Reply:
i think my question caused you removing this. but it was just meant as a question. i think that should be reintroduced. sorry for the confusion 😬